setup.py

```diff
@@ -2,19 +2,42 @@
 
 from setuptools import setup
 
+import i18n
```

Comment:
BTW: other techniques that don't have this fragility: https://packaging.python.org/single_source_version/